client/src/components/RelatedProducts/Cards/AllCards.jsx

```diff
@@ -1,41 +1,43 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Individualcard, IndCard } from './Individualcard';
-import ProductImg from './ProductImg';
-import sampleRelatedData from './sampleRelatedData';
-import sampleRelatedThumbnails from './sampleRelatedThumbnails';
+import { Individualcard } from './Individualcard';
+import { useProd } from './ProdContext';
+import { useThumbnail } from './thumbnailContext';
+
+function AllCards() {
+  const relatedProds = useProd();
+  const thumbnail = useThumbnail();
+
+  const zippedArray = [];
+  if (!thumbnail.data) {
+    console.log('loading thumbnail');
+  } if (!relatedProds.data) {
+    console.log('loading related prods');
```

Comment:
Could you comment out the console.logs before merging?

Reply:
Fixed!